core/blockchain.go

```diff
@@ -1824,6 +1824,7 @@ func (bc *BlockChain) InsertChain(chain types.Blocks) (int, error) {
 // completes, then the historic state could be pruned again
 // TODO(GalaIO): if enable cancun, it must set received blob cache for check, remove cache when failed
 func (bc *BlockChain) insertChain(chain types.Blocks, setHead bool) (int, error) {
+	// todo 4844 after cancun the blocks should have sidecars!
```

Comment:
The 4844 importer PR has added DA check in this function, the todo could be removed.